renderer/components/IntlProvider.js

```diff
@@ -13,6 +13,8 @@ require('@formatjs/intl-displaynames/locale-data/en')
 // https://formatjs.io/docs/react-intl/components#rawintlprovider
 const cache = createIntlCache()
 
+const localesWithRTL = ['ar', 'fa']
```

Comment:
We might as well add all the possible RTL language codes in here: https://github.com/shadiabuhilal/rtl-detect/blob/master/lib/rtl-detect.js#L138